src/clients/client.ts

```diff
@@ -184,6 +185,19 @@ class Client {
     }
   }
 
+  /**
+   * Swap two indexes
+   *
+   * @memberof MeiliSearch
+   * @method swapIndexes
+   * @param {IndexesSwapParams} queryParam - Indexes to swap
+   * @returns {Promise<Task>} - Promise returning a task
+   */
+  async swapIndexes(indexes: IndexesSwapParams): Promise<EnqueuedTask> {
```

Comment:
Inconsistency detected between the doc and the param name, `queryParam`/`indexes`.